pkg/certmanager/certmanager.go

```diff
@@ -99,14 +99,14 @@ func (c *CertManager) EnsureCertificate(vol *csiapi.MetaData, keyBundle *util.Ke
 
 	certPath := util.CertPath(vol)
 
-	if err := util.WriteFile(certPath, cr.Status.Certificate, 0600); err != nil {
+	if err := util.WriteFile(certPath, cr.Status.Certificate, 0644); err != nil {
```

Comment:
Similar, but why does owner vs group have different permissions?

Reply:
I am not an expert on these file permissions, but [secrets-store-csi-driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) uses [0644](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/23e09d50f85e45440d105711f994b5fc7350cbf2/pkg/secrets-store/nodeserver.go#L55).